### ISSUE TITLE
onlykey-gent: fails at runtime due to missing Cython

### ISSUE THREAD

**Issue by adrian-gierakowski** [NixOS/nixpkgs#325546]
### Describe the bug
Running:

```sh
onlykey-agent --version
```

crashes with error:

```
Traceback (most recent call last):
  File "/nix/store/84zr3y3k0s8446lgsjzvf08vlj1p9svs-onlykey-agent-1.1.15/bin/.onlykey-agent-wrapped", line 9, in <module>
    sys.exit(ssh_agent())
             ^^^^^^^^^^^
  File "/nix/store/84zr3y3k0s8446lgsjzvf08vlj1p9svs-onlykey-agent-1.1.15/lib/python3.11/site-packages/onlykey_agent/__init__.py", line 5, in <lambda>
    ssh_agent = lambda: libagent.ssh.main(DeviceType)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/46crk8s8m4cmnbwvqkrf6pv7gqhc7xg0-python3.11-libagent-1.0.6/lib/python3.11/site-packages/libagent/ssh/__init__.py", line 185, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/46crk8s8m4cmnbwvqkrf6pv7gqhc7xg0-python3.11-libagent-1.0.6/lib/python3.11/site-packages/libagent/ssh/__init__.py", line 268, in main
    args = create_agent_parser(device_type=device_type).parse_args()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/46crk8s8m4cmnbwvqkrf6pv7gqhc7xg0-python3.11-libagent-1.0.6/lib/python3.11/site-packages/libagent/ssh/__init__.py", line 75, in create_agent_parser
    resources_map = {r.key: r for r in pkg_resources.require(agent_package)}
                                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/9ryla5qrjhmb3nix3cp0n57qx06scgc0-python3.11-setuptools-69.5.1/lib/python3.11/site-packages/pkg_resources/__init__.py", line 937, in require
    needed = self.resolve(parse_requirements(requirements))
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/9ryla5qrjhmb3nix3cp0n57qx06scgc0-python3.11-setuptools-69.5.1/lib/python3.11/site-packages/pkg_resources/__init__.py", line 798, in resolve
    dist = self._resolve_dist(
           ^^^^^^^^^^^^^^^^^^^
  File "/nix/store/9ryla5qrjhmb3nix3cp0n57qx06scgc0-python3.11-setuptools-69.5.1/lib/python3.11/site-packages/pkg_resources/__init__.py", line 839, in _resolve_dist
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'Cython>=0.23.4' distribution was not found and is required by onlykey
```

### Steps To Reproduce
1. nix run nixpkgs#onlykey-agent -- --version

### Expected behavior
Should print out the version without crashing

### Additional context
following patch fixes it:

```patch
diff --git a/pkgs/tools/security/onlykey-agent/default.nix b/pkgs/tools/security/onlykey-agent/default.nix
index 36a1cb284759..51949fd91327 100644
--- a/pkgs/tools/security/onlykey-agent/default.nix
+++ b/pkgs/tools/security/onlykey-agent/default.nix
@@ -52,7 +52,7 @@ python3Packages.buildPythonApplication rec {
     sha256 = "sha256-SbGb7CjcD7cFPvASZtip56B4uxRiFKZBvbsf6sb8fds=";
   };
 
-  propagatedBuildInputs = with python3Packages; [ lib-agent onlykey-cli setuptools ];
+  propagatedBuildInputs = with python3Packages; [ lib-agent onlykey-cli setuptools cython_0 ];
 
   # move the python library into the sitePackages.
   postInstall = ''
``` 

### Notify maintainers

@kalbasit

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
output here
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by adrian-gierakowski** [comment]
happy to submit a PR

**Comment by kalbasit** [comment]
@adrian-gierakowski wasn't this solved by #325182?

**Comment by adrian-gierakowski** [comment]
> @adrian-gierakowski wasn't this solved by #325182?

I guess it was! I’ve updated my system just a couple of days before that commit but only noticed the issue this Monday.
